### PR TITLE
added location type as discussed in https://github.com/qwat/qwat-data-model/issues/36

### PR DESCRIPTION
**Pull request by tudorbarascu**
Do you want me to add a locationtype to the ui's ?

All the best,
Tudor

### CODE REVIEW

**Comment by tudorbarascu** [comment]
I want to also add an Emplacement tab in each ui, with a QListWidget.
Should look like this:
screenshot from 2015-06-11 08 27 57

Should I make a pull request with the ui's?

File: value_lists/vl_locationtype.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+INSERT INTO qwat_vl.locationtype (id,value_en,value_fr,value_ro) VALUES (9602,'unknown', 'inconnu', 'necunoscută');
+INSERT INTO qwat_vl.locationtype (id,value_en,value_fr,value_ro) VALUES (9603,'to be determined', 'à déterminer', 'de determinat');
+INSERT INTO qwat_vl.locationtype (id,value_en,value_fr,value_ro) VALUES (9604,'sidewalk','trottoir','trotuar');
+INSERT INTO qwat_vl.locationtype (id,value_en,value_fr,value_ro) VALUES (9605,'carriageable','carrossable','carosabil');
+INSERT INTO qwat_vl.locationtype (id,value_en,value_fr,value_ro) VALUES (9606,'cubic street stones','','piatră cubică');
+INSERT INTO qwat_vl.locationtype (id,value_en,value_fr,value_ro) VALUES (9607,'gravle','gravier','pietriş');
+INSERT INTO qwat_vl.locationtype (id,value_en,value_fr,value_ro) VALUES (9608,'sand','sable','nisip');
+INSERT INTO qwat_vl.locationtype (id,value_en,value_fr,value_ro) VALUES (9609,'soil','terre','pământ');
+INSERT INTO qwat_vl.locationtype (id,value_en,value_fr,value_ro) VALUES (9610,'concrete','béton','beton');
+INSERT INTO qwat_vl.locationtype (id,value_en,value_fr,value_ro) VALUES (9611,'open space','','spaţiu verde');
```
Comment (3nids) [inline value_lists/vl_locationtype.sql]: what do you mean by open space?

Comment (tudorbarascu) [inline value_lists/vl_locationtype.sql]: http://en.wikipedia.org/wiki/Urban_open_space

Basically, green space/open space/ espace vert. Places where there are no constructions inside the city with trees/bushes.. etc.

**Comment by 3nids** [comment]
ok for UI.
For the french term, go for "espace vert"

**Comment by tudorbarascu** [comment]
added "espace vert"

**Pull request merged**